test/src/main/java/org/apache/accumulo/test/functional/AccumuloClientIT.java

```diff
@@ -40,12 +42,28 @@
 import org.apache.accumulo.core.singletons.SingletonManager;
 import org.apache.accumulo.core.singletons.SingletonManager.Mode;
 import org.apache.accumulo.harness.AccumuloClusterHarness;
+import org.junit.Before;
 import org.junit.Test;
 
 import com.google.common.collect.Iterables;
 
 public class AccumuloClientIT extends AccumuloClusterHarness {
 
+  @Before
```

Comment:
It seems like it would be better to make this method an `@After` method.

Reply:
I agree.  I finally saw the error, the second time I ran the test:
org.apache.accumulo.core.client.AccumuloSecurityException: Error USER_EXISTS for user testuser - The user exists
But it would make more sense to put it as an `@After` method.

Reply:
agree.  I'll test it out. BUT what if we updated the harness to create users prefixed with the methodname (like the tables) - then the deleteusers in the harness would work.  Is there a limit to the number of characters a username can have?

Reply:
> what if we updated the harness to create users prefixed with the methodname (like the tables)

That may make sense to do.  That feels like a follow on issue, what do you think?

The class name as a prefix is confusing because with inheritance and multiple classes its hard to know if you got it right.  Maybe just use a constant like 

```java
public static final String HARNESS_USER_PREFIX="a1b1ae6bfef2ca11d92b86ab60930b3c56971f6e"
```

Then all code can just use that constant.  I picked the latest commit hash on the master branch for the value.

Reply:
So I see this request is accepted.  Do you still want me to move the deleteuser to the @After?  Or did someone do that already?

Reply:
I think it would be good to make it after